core/lib/brightcontent/core.rb

```diff
@@ -1,3 +1,5 @@
+ENV['RANSACK_FORM_BUILDER'] ||= '::SimpleForm::FormBuilder'
```

Comment:
I.e. By default the `search_form` helper yields an instance of a Ransack specific form builder on which no SimpleForm methods are available. When this ENV var is set Ransack's form builder class will extend SimpleForm's form builder class, making available SimpleForm's form helpers (among which is `#input`, the one we're using).